src/main.rs

```diff
@@ -20,11 +20,10 @@ mod vulnerability;
 
 use bevy::{app::AppExit, prelude::*, window::WindowMode};
 #[cfg(debug_assertions)]
-use bevy_inspector_egui::WorldInspectorPlugin;
-use bevy_kira_audio::AudioPlugin;
+//use bevy_inspector_egui::WorldInspectorPlugin;
```

Comment:
Why did you remove the debug options?

Reply:
Answered on discord : Disabling egui for now in order to migrate progressively to 0.6